guides/shop-managers-guide/77-configuring-address-validation.md

```diff
@@ -0,0 +1,12 @@
+# Address validation
+
+You can find the settings for the for the Address Validation under Settings -> Address Validation in the Admin.
+![Address validation](_assets/77-address-validation-settings.png)
+
+In the settings you can see a table which displays the list of address validation services that have been set up and the country to which it applies.
+
+The Add Address Validation Services button provides a modal to add more services to the list.
```

Comment:
To add a new address validation service, click **Add Address Validation Services** and fill in the information on the pop-up modal.